src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,8 +1,21 @@
 package mate.academy.hibernate.relations.model;
 
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "actors")
 public class Actor implements Cloneable {
+    @Id
+    @GeneratedValue (strategy = GenerationType.IDENTITY)
     private Long id;
     private String name;
+    @ManyToOne (fetch = FetchType.EAGER)
```

Comment:
ManyToOne has fetch type 'Eager' by default
```suggestion
    @ManyToOne
```